src/browser/BrowserService.cpp

```diff
@@ -1348,6 +1348,22 @@ QList<Entry*> BrowserService::getPasskeyEntries(const QString& rpId, const Strin
     return entries;
 }
 
+// Returns all Passkey entries for the current Relying Party and identical user handle
+QList<Entry*> BrowserService::getPasskeyEntriesWithUserHandle(const QString& rpId,
+                                                              const QString& userId,
+                                                              const StringPairList& keyList)
+{
+    QList<Entry*> entries;
+    for (const auto& entry : searchEntries(rpId, "", keyList, true)) {
+        if (entry->hasPasskey() && entry->attributes()->value(BrowserPasskeys::KPEX_PASSKEY_RELYING_PARTY) == rpId
+            && entry->attributes()->value(BrowserPasskeys::KPEX_PASSKEY_USER_HANDLE) == userId) {
```

Comment:
Do we want to do a Qt::CaseInsensitive compare instead? May also want to trim the userId of prefix and suffix whitespace. Just thinking of some contingencies based on values users type in.

Reply:
These are values that users do not add by themselves.

Reply:
Ok gotcha, can ignore